docs/changelog/0.13.0.mdx

```diff
@@ -33,6 +33,31 @@ WITH (key_field='id');
 - Support for `CREATE INDEX CONCURRENTLY`
 - Support for indexing partitioned tables
 
+### Deprecated Syntax Migration
```

Comment:
I added this back here since it was requested by a customer to help find how to upgrade